CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+    
+Even if some of these dependencies are not in active use, the git "update" command may fail to execute if entries and URLs for **all submodule** dependencies cloned from your fork are not found in the .gitmodules file.
+
+    [submodule "Submodules/x360ce"]
+	    path = Submodules/x360ce
+	    url = https://github.com/lucasassislar/x360ce
+
+    [submodule "Submodules/ProtoInput"]
+        path = Submodules/ProtoInput
+        url = https://github.com/ilyaki/protoinput
+
+    [submodule "Submodules/nukeupdater"]
+        path = Submodules/nukeupdater
+        url = https://github.com/lucasassislar/nukeupdater
+        
+    [submodule "Submodules/TypescriptSyntaxPaste"]
+        path = Submodules/TypescriptSyntaxPaste
+        url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git
+
+#### Project Branches
+As previously mentioned, the "master" branch of the project is the primary development branch of Nucleus Coop. Generally, you should be safe to assume that this is the current branch for everyone to work with. If you ever see other branches with recent updates and aren't sure which branch you should be forking or working from, just ask!
+
+When you submit a Pull Request, make _sure_ that you have the correct "merge" target set. The "base repository" is the target _destination_ for your changes, while the "head repository" is the _source_ of the changes (i.e. your forked repository).
+
+The current iteration of the project has gone through several forks in its history, and when you go to submit a pull request, GitHub may default the merge target to a _different_ fork.
+
+**After clicking "New Pull Request" on the Pull Request page, make sure that your base and head repositories are correct before continuing.**
+
+Make sure that for the "base" repository, the "master" branch is selected. If you are merging to a different branch for some reason, merge to that branch instead. For the head, the branch should be whatever branch you would like to merge from in your own repository fork.
+
+_The **Comparing Changes** selectors must display as follows for the Pull Request to work correctly._
+
+**Base Repository**
+>base repository: SplitScreen-Me/splitscreenme-nucleus
+
+If Github defaults the head to SplitScreen-Me/splitscreenme-nucleus, you will need to change the head _first_. Otherwise, GitHub may see you as trying to merge from within the same project and drop the option to select target/destination forks (requiring you to backtrack and start base/head selection over again).
+
+**Base**
+>base:master
+
+Use "master" unless you are intentionally merging with a different branch for some reason (in which case, use the target branch instead).
+
+**Head Repository**
+>head repository: YourUserName/ForkRepoName
+
+**Compare**
+>compare: YourSourceBranch
+

```

Comment:
This section could probably be shortened quite a bit, though I'm not sure exactly how 😅

Reply:
Yeah, I was kinda iffy on leaving this one as-is, since it's kinda long, lol. If we're operating on the assumption that the contributor knows how to use Github and do a Pull Request, we could probably shave off all the detailed parts and just leave the comments highlighting that it might default the base to an older fork of the project (and include the reminder about changing the head first to avoid having to do it all again).

Reply:
New commits should have a heavily simplified version of this section. May still be a bit of redundancy with the Pull Request part of the article. I'm not _entirely_ sure it's absolutely necessary, since anyone who knows how to use GitHub should probably be able to notice that when the head/base are wrong, lol.

Maybe just take everything from (in the new version) "GitHub may default..." on down and put it in the Pull Request section instead? And just leave this section for a quick note/clarification on the project branches (giving a space to edit in details later in case the current "master" paradigm ever changes).

Reply:
Implemented a first pass of a hatchet-job to this section, but may want to follow-up on my previous comment and further trim it/restructure it, depending on what everyone thinks.

Reply:
Imo it looks okey now. I'll resolve this and make another suggestion related to this in a new review :)